azurerm/internal/services/storage/resource_arm_storage_account.go

```diff
@@ -452,6 +453,27 @@ func resourceArmStorageAccount() *schema.Resource {
 				},
 			},
 
+			"static_website": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
since we'll only flatten a `static_website` block when there's one enabled we can remove Computed here (since there'll be 0 blocks when omited)

```suggestion
```